### PR TITLE
Feature/Add instructor directions

### PR DESCRIPTION
**Pull request by jkeam**
# Feature - Add Instructor Directions

## What
This PR adds instructor setup instructions for the Serverless workshop, mainly around creating a central Kafka cluster that all workshop participants can share.

## Why
The concern is that having each student spin up their own cluster uses too many resources.

## How
Updated `README.md` instructions that the workshop instructor must do before the workshop.

## Who
This content is for workshop instructors.

### CODE REVIEW

**Review by theckang — APPROVED** [review]

**Review by theckang — DISMISSED** [review]

**Review by theckang — CHANGES_REQUESTED** [review]
In the Kafka lab, there are many steps where the participant user will view api-resources or pods in other namespaces they don't have access to by default.  For example, `kafka`, `knative-sources`, `knative-eventing`, are some of the ones I saw.

We'll need to add a step to grant RBAC roles to workshop users to access these.  Here's an example: https://github.com/RedHatGov/serverless-workshop-code/blob/main/setup/rbac.yaml

File: kafka/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Kafka
+
+This section contains the necessary resources to be able to setup a Kafka Cluster and Topic.  The idea is that each participant would set up their own cluster and topic.
```
Comment (theckang) [inline kafka/README.md]: each participant would use a shared cluster and create their own topics.

**Comment by jkeam** [comment]
> In the Kafka lab, there are many steps where the participant user will view api-resources or pods in other namespaces they don't have access to by default. For example, `kafka`, `knative-sources`, `knative-eventing`, are some of the ones I saw.
> 
> We'll need to add a step to grant RBAC roles to workshop users to access these. Here's an example: https://github.com/RedHatGov/serverless-workshop-code/blob/main/setup/rbac.yaml

I'm not sure, so please correct me, but I thought all workshop users created by the workshop (rhpds) were admins?

**Comment by theckang** [comment]
> > In the Kafka lab, there are many steps where the participant user will view api-resources or pods in other namespaces they don't have access to by default. For example, `kafka`, `knative-sources`, `knative-eventing`, are some of the ones I saw.
> > We'll need to add a step to grant RBAC roles to workshop users to access these. Here's an example: https://github.com/RedHatGov/serverless-workshop-code/blob/main/setup/rbac.yaml
> 
> I'm not sure, so please correct me, but I thought all workshop users created by the workshop (rhpds) were admins?

They are not.  Only `opentlc-mgr` is an admin.  This is also what we want.  It's a big risk to give everyone admin access to the cluster.

As an reference point - workshop users in the istio workshop only have view access to `istio-system` and admin access to their user project `userx`.

**Comment by jkeam** [comment]
@theckang Ready for another round of review.  I changed a few steps to use the new custom OpenShift `KnativeKafka` resource.

**Comment by theckang** [comment]
@jkeam `kafka-knative.yml` is missing

**Comment by jkeam** [comment]
> @jkeam `kafka-knative.yml` is missing

Brain fart 🧠  I added it.

File: README.md [hunk 2]
```diff
@@ -27,21 +27,7 @@ oc apply -f ./setup/serving.yml
```
Comment (theckang) [inline README.md:27]: under `3. Create the knative-eventing namespace...`

it should be `oc apply -f ./setup/eventing.yml`

**Review by theckang — CHANGES_REQUESTED** [review]
Minor typo

File: README.md (hunk 2 above)
Comment (jkeam) [reply README.md:27]: fixed

**Review by theckang — APPROVED** [review]

**Pull request merged**